src/Api/AdminConsole/Controllers/OrganizationDomainController.cs

```diff
@@ -133,6 +135,20 @@ public async Task<OrganizationDomainSsoDetailsResponseModel> GetOrgDomainSsoDeta
         return new OrganizationDomainSsoDetailsResponseModel(ssoResult);
     }
 
+    [AllowAnonymous]
```

Comment:
Can you explain to me why these need `AllowAnonymous`? I can see it on the method above too, but I don't immediately understand why. Claiming domains is done from authorized sessions in the admin console, isn't it?

Reply:
This is a get for retrieving the claimed domains. It is used when attempting to log in via SSO. The user is not logged in at this point. This endpoint is retrieved in order to try to determine the org and domain needed for SSO login.